src/App.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Layout from './components/layout/Layout';
+import Home from './components/home/Home';
+import Doctor from './components/details/Details';
+import SignUp from './components/auth/SignUp';
+import SignIn from './components/auth/SignIn';
+import Bookings from './components/bookings/Bookings';
+import Book from './components/book/Book';
+import Account from './components/account/Account';
+import Doctors from './components/doctors/Doctors';
+import DoctorNew from './components/doctors/doctor-new/DoctorNew';
+import DoctorEdit from './components/doctors/doctor-edit/DoctorEdit';
+
+const App = () => (
```

Comment:
Organize the routes in a logical order that follows the user's journey through the application. This makes it easier for other developers to understand how the application works.
And used comments to document their purpose.
E.g:
{/* Doctor Routes */} <Route path="doctors" element={<Doctors />} /> <Route path="doctors/new" element={<DoctorNew />} /> <Route path="doctors/:id" element={<Doctor />} /> <Route path="doctors/:doctorId/edit" element={<DoctorEdit />} />